ftw/linkchecker/command/checking_links.py

```diff
@@ -322,7 +321,24 @@ def send_mail_with_excel_report_attached(email_address, plone_site_obj,
     portal = api.content.get(plone_site_path)
     report_mailer_instance = report_mailer.MailSender(portal)
     report_mailer_instance.send_feedback(
-        email_subject, email_message, email_address, xlsx_file)
+        email_subject, email_message, email_address, xlsx_file, file_name)
+
+
+def upload_report_to_filelistingblock(filelistingblock_url, xlsx_file, file_name):
+    portal = api.portal.get()
+    try:
+        file_listing_block = portal.unrestrictedTraverse(path=filelistingblock_url.encode('utf-8'))
+    except Exception as e:
+        # TODO: log exception (this is most likely be raised if
+        # the path to a file listing block is wrong.
+        raise
+
+    xlsx_file.seek(0)
+    data = xlsx_file.read()
+
+    file = plone.api.content.create(container=file_listing_block, type='File', title=file_name, file=data)
```

Comment:
Don't override pythons `file` here. You could simply use `file_`, oder `file_content_type`